apps/web/src/pages/settings/tabs/EmailSettings.tsx

```diff
@@ -108,11 +108,10 @@ export const EmailSettings = () => {
                 data-test-id="inbound-parse-domain"
                 disabled={readonly}
                 value={field.value || ''}
-                label={'Domain routing'}
+                label={'Whitelisted Domain'}
```

Comment:
```suggestion
                label={'Allowed Domains'}
```

What do you think about this more inclusive terminology?

Reply:
I am good with this one